components/plugins/wordpress/WordpressSteps.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <section class="container py-5 mt-3">
+    <h2 class="section__heading text-center">
+      Steps to Embed a Formester Form
+    </h2>
+
+    <div class="row steps-container px-3 py-5 d-flex flex-column flex-lg-row">
+      <div
+        class="col d-flex flex-column-reverse align-items-center justify-content-center"
+      >
+        <div class="mt-5">
+          <span class="step__heading">Step 1</span>
+          <h3 class="step__section-heading">Install Formester Plugin</h3>
+          <p class="step__section-description">
+            Visit
+            <a
+              href="WordPress.org"
+              target="_blank"
+              class="text-decoration-underline"
+              >WordPress.org</a
+            >
```

Comment:
In href if you have to add the complete link to wordpress i.e. `href="https://wordpress.com/"`